contracts/strategies/PermissionedStrategy.sol

```diff
@@ -90,6 +101,7 @@ contract PermissionedStrategy is IStrategy, AccessControl {
         }
 
         // 3. perform transfer
+        totalRequestedWithdraws -= amount;
```

Comment:
```suggestion
        // amount <= totalRequestedWithdraws
        unchecked {
            totalRequestedWithdraws -= amount;
        }
```
Also should this be `totalRequestedWithdraws -= amountToTransfer;`?


(The same check can be added for `uint256 outstandingAmount = amount - currentBalance;`, since `amount > currentBalance`)